src/routes/publisherRoutes.ts

```diff
@@ -59,12 +59,15 @@ router.post(
       req.session.uploadErrors = errs;
       return res.redirect("/publish/csv/upload-summary");
     } catch (err) {
-      console.error(err);
```

Comment:
We probably still want to print the error in case it's something that needs fixing?